### PR TITLE
Improve and update windows text in INSTALL.md

### PR DESCRIPTION
**Pull request by ChrisJefferson**
The cygwin related text in INSTALL.md is very old, and mostly useless. Overhaul this now.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -563,23 +563,21 @@ fix is known. (One can type `POW(a,b)` for `a^b`.)
 
 ## Problems specific to Windows
 
-### Cut and Paste does not work
+The best way to use GAP in Windows is to use the "Windows Subsystem for Linux",
+also known as WSL. This can be found in the "Microsoft Store" in Windows, and
+allows you to install a full copy of Ubuntu. You can then follow the guidance
+for building and using GAP in Linux inside WSL.
```
Comment (fingolfin) [inline INSTALL.md:569]: Perhaps include a link to instructions like https://learn.microsoft.com/en-us/windows/wsl/install

File: INSTALL.md [hunk 2]
```diff
@@ -563,23 +563,21 @@ fix is known. (One can type `POW(a,b)` for `a^b`.)
...
-GAP cannot be started in the Windows Command Prompt shell (via `gapcmd.bat`)
-in the remote desktop. To start GAP in the remote desktop, use the script
-`gap.bat` which should work in such setting.
-
-### You get an error message about the `cygwin1.dll`
-
-GAP comes with a version of this dynamic library. If you have another
-version installed (use "Find"), delete the older one (and probably copy the
-newer one in both places).
+The windows release can be installed globally for all users, or just for the
```
Comment (fingolfin) [inline INSTALL.md:578]:
```suggestion
The Windows release can be installed globally for all users, or just for the
```

File: INSTALL.md [hunk 3]
```diff
@@ -558,28 +558,27 @@ This is a problem if you are running a keyboard driver for some non-english
...
+how to install readline and section 5 on how to recompile GAP, for Windows no
 fix is known. (One can type `POW(a,b)` for `a^b`.)
 
 ## Problems specific to Windows
 
-### Cut and Paste does not work
+The best way to use GAP in Windows is to use the "Windows Subsystem for Linux"
+( https://learn.microsoft.com/en-us/windows/wsl/install ), also known as WSL.
+This can be found in the "Microsoft Store" in Windows, and allows you to install
+a full copy of Ubuntu. You can then follow the guidance for building and using
```
Comment (fingolfin) [inline INSTALL.md:569]:
```suggestion
a full copy of Linux. You can then follow the guidance for building and using
```

File: INSTALL.md [hunk 4]
```diff
@@ -558,28 +558,27 @@ This is a problem if you are running a keyboard driver for some non-english
...
 please refer to the section "GAP for macOS" below for information on
-how to install readline and section 5 on how to recompile GAP, for windows no
+how to install readline and section 5 on how to recompile GAP, for Windows no
 fix is known. (One can type `POW(a,b)` for `a^b`.)
 
 ## Problems specific to Windows
 
-### Cut and Paste does not work
+The best way to use GAP in Windows is to use the "Windows Subsystem for Linux"
+( https://learn.microsoft.com/en-us/windows/wsl/install ), also known as WSL.
```
Comment (fingolfin) [inline INSTALL.md:567]:
```suggestion
(<https://learn.microsoft.com/en-us/windows/wsl/install>), also known as WSL.
```

File: INSTALL.md [hunk 5]
```diff
@@ -558,28 +558,27 @@ This is a problem if you are running a keyboard driver for some non-english
...
+This can be found in the "Microsoft Store" in Windows, and allows you to install
+a full copy of Ubuntu. You can then follow the guidance for building and using
+GAP in Linux inside WSL.
 
-You might want to try different shells, using both of the two `.bat`
-files in the `bin` directory: `gap.bat` and `gapcmd.bat`. Also,
-<https://www.gap-system.org/Faq/faq.html#4> might give a remedy.
+We also support building GAP using 'Cygwin', which is a UNIX wrapper for
+Windows. Cygwin is used for making Windows release. Almost all packages are
+supported in the Windows releases.
```
Comment (fingolfin) [inline INSTALL.md:574]: I feel "Windows release" could be misunderstood. Maybe like this
```suggestion
Windows. Cygwin is used for making the Windows installers for GAP.
Almost all packages are supported in these GAP releases for Windows.
```
Although I wonder about that final sentence, what is it meant to convey to the user? It seems to presuppose that users will assume some packages won't work, but new user have no reason to do that, to the contrary; so I think this sentence will primarily trigger the reaction "wait, some packages are not supported?! Which? And why?"

So maybe just drop that sentence, or else expand it?

File: INSTALL.md [hunk 6]
```diff
@@ -558,28 +558,27 @@ This is a problem if you are running a keyboard driver for some non-english
...
-in the remote desktop. To start GAP in the remote desktop, use the script
-`gap.bat` which should work in such setting.
-
-### You get an error message about the `cygwin1.dll`
-
-GAP comes with a version of this dynamic library. If you have another
-version installed (use "Find"), delete the older one (and probably copy the
-newer one in both places).
+The Windows release can be installed globally for all users, or just for the
+current user. This is the best way to install GAP on a machine where you do
```
Comment (fingolfin) [inline INSTALL.md:580]:
```suggestion
current user. The latter is the best way to install GAP on a machine where you do
```

File: INSTALL.md [hunk 7]
```diff
@@ -558,28 +558,27 @@ This is a problem if you are running a keyboard driver for some non-english
...
+The best way to use GAP in Windows is to use the "Windows Subsystem for Linux"
+( https://learn.microsoft.com/en-us/windows/wsl/install ), also known as WSL.
+This can be found in the "Microsoft Store" in Windows, and allows you to install
+a full copy of Ubuntu. You can then follow the guidance for building and using
+GAP in Linux inside WSL.
 
-You might want to try different shells, using both of the two `.bat`
-files in the `bin` directory: `gap.bat` and `gapcmd.bat`. Also,
-<https://www.gap-system.org/Faq/faq.html#4> might give a remedy.
+We also support building GAP using 'Cygwin', which is a UNIX wrapper for
```
Comment (fingolfin) [inline INSTALL.md:572]: Note that we write "Unix" not "UNIX" elsewhere in this document, maybe align it?

Also, "wrapper" an adequate term here? Perhaps like this?
```suggestion
We also support building GAP using 'Cygwin', which emulates a Unix environment on
```

File: INSTALL.md [hunk 8]
```diff
@@ -558,28 +558,27 @@ This is a problem if you are running a keyboard driver for some non-english
...
 accent and do not pass it properly to GAP. For macOS users, as a workaround
 please refer to the section "GAP for macOS" below for information on
-how to install readline and section 5 on how to recompile GAP, for windows no
+how to install readline and section 5 on how to recompile GAP, for Windows no
 fix is known. (One can type `POW(a,b)` for `a^b`.)
 
 ## Problems specific to Windows
 
-### Cut and Paste does not work
+The best way to use GAP in Windows is to use the "Windows Subsystem for Linux"
```
Comment (fingolfin) [inline INSTALL.md:566]: Elsewhere in this file we write "If you are using the `.exe` installer (which we strongly recommend),"

Which sounds as if it contradicts this. Well, it doesn't really (that sentence was meant to be in contrast to using the `.zip` file version, which we don't provide anymore I think), but a fresh reader can't know that. I guess we should adjust that other place then perhaps?

**Comment by fingolfin** [comment]
There is more outdated text specific to Windows in the section "Problems specific to Windows", e.g. references to `gap.bat` and `gapcmd.bat` or to FAQ section 4 which no longer exists.

File: INSTALL.md (hunk 3 above)
Comment (ChrisJefferson) [reply INSTALL.md:569]: I mean ubuntu here, as that's actually what you have to go look for (Ubuntu), and Ubuntu is by far the best supported OS in WSL (I've tried others and they are always fragile).

File: INSTALL.md (hunk 5 above)
Comment (ChrisJefferson) [reply INSTALL.md:574]: Will drop it -- older users might expect many packages to not work (as they didn't used to), but I agree ther eis no need to say that here.

**Comment by ChrisJefferson** [comment]
I made another sweep through, and tried to improve all windows related test. I decided to reduce how much we sell WSL, because for "beginner users" they are probably best with the standard installer, as they might not want to get involved with the command line.

File: INSTALL.md (hunk 3 above)
Comment (fingolfin) [reply INSTALL.md]: Then maybe explain it as such? The text as written is ok for someone who already knows what Linux and Ubuntu are, but my experience is that an average Windows user does not understand how these things relate...

**Review by fingolfin — APPROVED** [review]

**Pull request merged**